src/module/data/item/ability.mjs

```diff
@@ -29,6 +29,7 @@ export default class AbilityModel extends BaseItemModel {
 
     schema.keywords = new fields.SetField(new fields.StringField({required: true, blank: false}));
     schema.type = new fields.StringField({required: true, blank: false, initial: "action"});
+    schema.category = new fields.StringField({required: true, blank: true, initial: ""}),
```

Comment:
```suggestion
    schema.category = new fields.StringField({required: true, nullable: false}),
```

Blank is true by default, and StringField has some smart initial handling if you throw in the nullable: false